main.go

```diff
@@ -285,3 +318,37 @@ func main() { //nolint:funlen
 		os.Exit(1)
 	}
 }
+
+func createSecretCacheConfig(platform cluster.Platform) map[string]cache.Config {
+	namespaceConfigs := map[string]cache.Config{
+		"istio-system":      {FieldSelector: fields.Set{"metadata.name": "knative-serving-cert"}.AsSelector()}, // for expiration case
+		"openshift-ingress": {},
+	}
+	switch platform {
+	case cluster.ManagedRhods:
+		namespaceConfigs["redhat-ods-monitoring"] = cache.Config{}
+		namespaceConfigs["redhat-ods-applications"] = cache.Config{}
+	case cluster.SelfManagedRhods:
+		namespaceConfigs["redhat-ods-applications"] = cache.Config{}
+	default:
+		namespaceConfigs["opendatahub"] = cache.Config{}
+	}
+	return namespaceConfigs
+}
+
+func createDeploymentCache(platform cluster.Platform) map[string]cache.Config {
```

Comment:
createDeploymentCacheConfig as above? Since it doesn't create the Cache, but just its config.

Reply:
thought i updated it to setDeploymentCache last week? or i did not push it :thinking:

Reply:
seems, it does only live in my local.............